vuepress/docs/next/docs/getting-started/development-concepts.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebarDepth: 2
+---
+
+# Development Concepts on Entando
+ 
+On Entando Composition Platform, the development of an application is broken down into modular steps of create, curate, compose, and consume. In the simplest terms, you start by **creating** functional building blocks, **curate** and organize the blocks in a catalogue, **compose** them on a page or website, which is then **consumed** in an app where it can be scaled and improved upon. 
+
+Entando provides a composable environment where these building blocks can be built independently, enabling parallel teams to accelerate development and innovation on a cloud and Kubernetes-native stack.
+
+![Entando Platform Development Process](./img/concepts.png)
+
+## Create Components 
+The first step in building a modern application on Entando is to design and create each independant component or functional block for your application. Developers can use Entando Component Generator or Blueprint to automate the building and packaging of project files, with the flexibility to customize. In this way, new business capabilities are integrated into your application without overhauling existing systems. 
+
+For more information on how to Create Components on Entando, see the docs below or follow a tutorial:
+
+| Documentation | Tutorials        |
+| :------------ | :--------- |
+| [Entando Component Generator](../create/component-gen-overview.md) | [Micro Frontends](../../tutorials/create/mfe/) |
+| [Entando Blueprint Feature](../create/blueprint-features.md) | [Microservices](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) |
+| [Component Generation Technologies](../create/component-gen-tech.md) | [Build and Publish Project Bundles](../../tutorials/create/pb/publish-project-bundle.md) |
+
+
+	
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means developers manage a central catalog of components and share it across an organization. Entando's distributed and composable development process allows you to quickly edit, test, analyze, and publish a singlular component or packaged business capability (PBC) repeatedly, without touching other parts of your application. This includes the ability to guide content, versioning, and metadata through the use of bundles. 
```

Comment:
"developers manage a central catalog.." -> "developer teams can share components within a central catalog made available to multiple teams or organizations"
"singlular" -> singular
"guide content..." -> "publish content and pages along with functional components such as micro frontends and microservices."